src/views/StatsView.vue

```diff
@@ -61,9 +61,7 @@
         </p>
       </div>
       <!-- Rankings Table -->
-      <template
-        v-if="selectedSeason && selectedSeason.rankings && selectedSeason.rankings.length > 0"
-      >
+      <template v-if="selectedSeason && selectedSeason.rankings && selectedSeason.rankings.length > 0">
```

Comment:
This should be converted to a computed prop since you have so many conditionals in here.